srv/InvokeStateChange.srv

```diff
@@ -0,0 +1,9 @@
+# Invoke State Change
+# Updates PackML use the desired state as the "current" state
+# Note: This violates the state machine, and will update timers only.
+#   It will not advance the underlying FSM state. Future events may trigger abnormal behavior.
+
+string name       # Name of the desired state
+int32 state_enum  # Enumeration of the desired state (See State.msg)
```

Comment:
Does an enum need to be int32, or can we get by with int8?

Reply:
int32 was chosen to pass Simple Message semantics without type casting, but 32-bit is probably overkill.